packages/backend/src/core/FanoutTimelineEndpointService.ts

```diff
@@ -63,12 +64,20 @@ export class FanoutTimelineEndpointService {
 
 		const redisResult = await this.fanoutTimelineService.getMulti(ps.redisTimelines, ps.untilId, ps.sinceId);
 
+		// オプション無効時、取得したredisResultのうち、2つ以上ソースがあり、1つでも空であればDBにフォールバックする
+		let shouldFallbackToDb = ps.useDbFallback &&
+			(ps.preventEmptyTimelineDbFallback !== true && redisResult.length > 1 && redisResult.some(ids => ids.length === 0));
```

Comment:
デフォルト値がこっちのほうがわかりやすそう

```suggestion
		ps.preventEmptyTimelineDbFallback ??= false;
		let shouldFallbackToDb = ps.useDbFallback &&
			(!ps.preventEmptyTimelineDbFallback && redisResult.length > 1 && redisResult.some(ids => ids.length === 0));
```

あとはtrueのときに無効化するというのがちょっと分かりづらい気がするので
```suggestion
		ps.fallbackToDbIfHasEmptyTimeline ??= true;
		let shouldFallbackToDb = ps.useDbFallback &&
			(ps.fallbackToDbIfHasEmptyTimeline && redisResult.length > 1 && redisResult.some(ids => ids.length === 0));
```

とかのほうがいいのかもしれないと思いました

Reply:
このPR作成時に「中身が存在しないTLがある場合はDBを参照しに行く方が取得漏れがなくなるので妥当」という意味でprevent...をtrueにするという命名を行っていたけれどtrueの時に無効化はこの命名でも、紛らわしいか…？（ちょっと上の案、下の案、あるいはやらない、ほかの案などもそれぞれ考えてみます）